packaging/greenboot/functions.sh

```diff
@@ -18,10 +18,13 @@ OCGET_CMD="oc get ${OCCONFIG_OPT}"
 function print_boot_status() {
     # Source Greenboot configuration file if it exists
     local conf_file=/etc/greenboot/greenboot.conf
+    # shellcheck source=/dev/null
     [ -f "${conf_file}" ] && source ${conf_file}
 
-    local grub_vars=$(grub2-editenv - list | grep ^boot_ || true)
-    local boot_vars=$(set | egrep '^GREENBOOT_|^MICROSHIFT_' || true)
+    local grub_vars
+    local boot_vars
+    grub_vars=$(grub2-editenv - list | grep ^boot_ || true)
+    boot_vars=$(set | grep -E '^GREENBOOT_|^MICROSHIFT_' || true)
```

Comment:
Let's use read-only variables when possible.
```suggestion
    local -r grub_vars=$(grub2-editenv - list | grep ^boot_ || true)
    local -r boot_vars=$(set | grep -E '^GREENBOOT_|^MICROSHIFT_' || true)
```

Reply:
@eggfoobar , I'm not adding more comments / suggestions like this, but can we try to avoid splitting the variable declaration and definition by using read-only variables whenever possible?

Reply:
Sounds good, thanks @ggiguash , updated with your suggestions.